src/NuGetPackageManager/Converters/EmptyCollectionToVisibleConverter.cs

```
@@ -0,0 +1,35 @@
+﻿namespace NuGetPackageManager.Converters
+{
+    using Catel.MVVM.Converters;
+    using System;
+    using System.Collections;
+    using System.Globalization;
+    using System.Windows;
+    using System.Windows.Data;
+
+    [ValueConversion(typeof(ICollection), typeof(Visibility))]
+    public class EmptyCollectionToVisibleConverter : Catel.MVVM.Converters.IValueConverter
+    {
+        private static readonly CollectionToCollapsingVisibilityConverter _collectionToVisibility = new CollectionToCollapsingVisibilityConverter();
+
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            try
+            {
+                var visibility = (Visibility)_collectionToVisibility.Convert(value, targetType, parameter, culture);
+
+                return visibility == Visibility.Visible ? Visibility.Collapsed : Visibility.Visible;
+
+            }
+            catch (Exception)
```

never swallow exceptions in converters, behaviours, event handlers and other places which could cause an application crash.
just write error into the log.